CMakeLists.txt

```diff
@@ -45,8 +45,12 @@ include(cmake/Key4hepConfig.cmake)
 
 include(GNUInstallDirs)
 
-include(CTest)
 
 add_subdirectory(k4GaudiPandora)
 
+if(BUILD_TESTING)
+  include(CTest)
```

Comment:
`include(CTest)` creates the BUILD_TESTING option (defaults to ON).
So this should come before?

cf. https://cmake.org/cmake/help/latest/module/CTest.html

Reply:
Yes, fixed